samples/abn-sample/frontend/go/go.sum

```diff
@@ -0,0 +1,1897 @@
+bazil.org/fuse v0.0.0-20160811212531-371fbbdaa898/go.mod h1:Xbm+BRKSBEpa4q4hTSxohYNQpsxXPbPry4JJWOB3LB8=
```

Comment:
May not be required.

Reply:
is needed; Dockerfile does `go mod download`